CHANGES/1347.bugfix

```diff
@@ -0,0 +1,3 @@
+The signing task no longer signs cosign signatures, attestations and
```

Comment:
I just realized maybe providing a bit more info might be useful. How about: The pulp signing task that produces atomic type signature no longer....